guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
+angular.module('login', ['element', 'navigation']);
```

Comment:
Shouldn't this be not-here until we do get around to it?

That does sound familiar, but I also remember us deciding to always use "/" as a means of reaching home, even if home is not ultimately "/".